frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -123,32 +158,54 @@ const PaymentsSectionText = () => {
         />
       </>
     )
+  } else if (secretEnv === 'production') {
+    return (
+      <Skeleton isLoaded={!isLoading}>
+        <InlineMessage variant="info" my="2rem">
+          <Text>
+            Read{' '}
+            <Link isExternal href={GUIDE_PAYMENTS}>
+              our guide
+            </Link>{' '}
+            to set up a Stripe account. To enjoy bulk tender transaction rates,
+            send your Stripe account ID and raise a purchase order to Stripe.
+          </Text>
+        </InlineMessage>
+        <Checkbox
+          isChecked={disclaimerChecked}
+          mb="2rem"
+          onChange={(e) => setDisclaimerChecked(e.target.checked)}
+        >
+          I understand that if I do not send my Stripe account ID and raise a
+          purchase order to Stripe, I will be paying default transaction rates.
+        </Checkbox>
+        <StripeConnectButton isDisabled={!disclaimerChecked} />
+      </Skeleton>
+    )
+  } else {
+    return (
+      <Skeleton isLoaded={!isLoading}>
+        <InlineMessage variant="info" my="2rem">
+          <Text>
+            You are currently in test mode. You can choose to skip connecting a
+            Stripe account after clicking the button below.
+          </Text>
+        </InlineMessage>
+        <StripeConnectButton />
+      </Skeleton>
+    )
   }
-
-  return (
-    <Skeleton isLoaded={!isLoading} mb="2.5rem">
-      <Text>
-        Connect your Stripe account to this form to start collecting payments.
-      </Text>
-      <InlineMessage variant="info" mt="2rem">
-        <Text>
-          Don't have a Stripe account? Follow{' '}
-          <Link target="_blank" href={GUIDE_PAYMENTS}>
-            this guide
-          </Link>{' '}
-          to create one.
-        </Text>
-      </InlineMessage>
-    </Skeleton>
-  )
 }
 
 export const PaymentSettingsSection = (): JSX.Element => {
-  const { hasPaymentCapabilities } = useAdminFormPayments()
+  const { hasPaymentCapabilities, data } = useAdminFormPayments()
+  const stripeAccount = data?.account
   return (
     <>
       <PaymentsSectionText />
-      <StripeConnectButton />
+      {stripeAccount ? (
```

Comment:
Hmm most of the `StripeConnectButton` are within `PaymentsSectionText` right, should we just standardise and move this up? Inside of having a check and return `null` here

Reply:
yessir, as discussed offline, I'll refactor this part altogether